salt/modules/crmshmod.py

```
@@ -313,7 +315,11 @@ def _crm_init(
     if quiet:
         cmd = '{cmd} -q'.format(cmd=cmd)
 
-    return __salt__['cmd.retcode'](cmd)
+    return_code = __salt__['cmd.retcode'](cmd)
```

This looks quite dangerous. I'm wondering why `crm cluster init` should return an error code if we don't change the call. Do you mean for versions where multiple sbd devices are not supported?

What do you mean with your next comment?
sbd timeouts are not supported in any version